### PR TITLE
Fix loss for dealing with missing annotations.

### LINKED ISSUE #26: Mask loss fails on empty masks
The mask loss makes the training process crash if an image doesn't contain any annotations (and therefore has no masks).

### PR DESCRIPTION
**Pull request by enricoliscio**
The loss function returns 0 if there are no annotations, the usual loss function if there are.
Should fix #26.

### CODE REVIEW

File: keras_maskrcnn/losses.py [hunk 1]
```diff
@@ -3,75 +3,85 @@
 from . import backend
 
 
-def mask(iou_threshold=0.5, mask_size=(28, 28)):
-    def _mask(y_true, y_pred):
-        # split up the different predicted blobs
-        boxes = y_pred[:, :, :4]
-        masks = y_pred[:, :, 4:]
+def default_loss(y_true, y_pred, iou_threshold=0.5, mask_size=(28, 28)):
```
Comment (hgaiser) [inline keras_maskrcnn/losses.py]: Could we put this as a subfunction of `mask`, similar to `_mask` at the moment? Let's rename it to `_mask` and rename the current `_mask` to `_mask_conditional` (or something else if you have a better suggestion).

**Comment by hgaiser** [comment]
LGTM, merge it?
